ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/store-pick-up-email-template.html

```diff
@@ -1,7 +1,7 @@
 From:deepalia@hcl.com
 subject: Your HCLecomm.in order has been placed successfully
 
-Hi ${firstname} ${lastname},
+Hi,
```

Comment:
@anchal-a : why has first name and last name been removed ?

Reply:
Because As a guest user>store pickup, user is not filling his first and last name anywhere. So, in the e-mail it was coming as "Hi NA NA," . So we are removing it now.